lib/console/devices/devices.ex

```diff
@@ -114,23 +114,25 @@ defmodule Console.Devices do
 
   def create_device(attrs \\ %{}, %Organization{} = organization) do
     count = get_organization_device_count(organization)
+
     cond do
-      organization.name !== "Discovery Mode (Helium)" and count >= Application.get_env(:console, :max_devices_in_org) ->
```

Comment:
`organization.name !== "Discovery Mode (Helium)"` must stay if we're keeping disco mode code, right?

Reply:
oh just kidding, adding new function for create discovery device